### PR TITLE
Add onnxruntime and DLR canary test

### PR DESCRIPTION
**Pull request by frankfliu**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .github/workflows/canary-centos.yml [hunk 1]
```diff
@@ -18,6 +18,9 @@ jobs:
         uses: actions/setup-java@v1
         with:
           java-version: 1.8
+      - name: Install libgomp
```
Comment (stu1130) [inline .github/workflows/canary-centos.yml:21]: why do we need this

File: canary/build.gradle [hunk 2]
```diff
@@ -58,14 +63,35 @@ dependencies {
...
+        }
+        implementation "ai.djl.mxnet:mxnet-engine"
+        implementation "ai.djl.mxnet:mxnet-native-auto"
+    } else if (engine.startsWith("dlr")) {
+        implementation "ai.djl.dlr:dlr-engine"
+        if (engine.contains("-auto")) {
+            implementation "ai.djl.dlr:dlr-native-auto"
+        } else {
+            // TODO: remove version after DJL 0.10.0
+            implementation "ai.djl.dlr:${engine}:1.6.0:${os}-x86_64"
```
Comment (stu1130) [inline canary/build.gradle:84]: why not using dlr_version

File: .github/workflows/canary-centos.yml (hunk 1 above)
Comment (frankfliu) [reply .github/workflows/canary-centos.yml:21]: onnxruntime depends on libgomp, but not bundled with the jar. User has to manually install this dependency

File: canary/build.gradle (hunk 2 above)
Comment (frankfliu) [reply canary/build.gradle:84]: dlr_version is not defined in gradle.properties in the this project. And once we added to bom, we don't need the dlr_version any more.

**Review by stu1130 — APPROVED** [review]

**Pull request merged**